### ISSUE TITLE
give /teamstatus a bind, or build it into the UI

### ISSUE THREAD

**Issue by bmorel** [Unvanquished/Unvanquished#2073]
teamstatus (disabled by default) does not have a bind, nor is integrated in UI. I personally bound it to `F8` because it's far.

@cu-kai  and @necessarily-equal might be interested at implementing something there.

**Comment by slipher** [comment]
It would be weird to have a bind for it since it's an optional feature only available on a few servers. Not sure how to handle that.

**Comment by DolceTriade** [comment]
I question the utility of *always* having this info available...

**Comment by Gireen** [comment]
I also think there should be no default bind for this.

But displaying this information in the hud is a good thing and there is also an issue for that 
#1466

it would replace good team communication that you cant have in public games most of the time

**Comment by bmorel** [comment]
> It would be weird to have a bind for it since it's an optional feature only available on a few servers. Not sure how to handle that.

Agree, but my problem is that it gives "people who know" an edge on others, and can't really be discovered. I would like to make unvanquished more newcommer-friendly, and this means to reduce the number of "secret-features".
Reducing their number is basically: add a bind or integrate into HUD. Obviously, the problem with adding a bind is that most servers have it disabled (since this is an opt-in feature). At least there's an explicit message that the feature is disabled, though.

**Comment by slipher** [comment]
> Agree, but my problem is that it gives "people who know" an edge on others, and can't really be discovered. I would like to make unvanquished more newcommer-friendly, and this means to reduce the number of "secret-features".

Well this can all be perfectly accomplished by keeping the feature disabled server-side. The team status feature seems tailor-made for people who enjoy making fancy binds.
